docs/docs/0.4.0/documentation/http-api.md

```diff
@@ -1318,6 +1318,164 @@ Example:
 curl -XDELETE http://localhost:5681/meshes/mesh-1/traffic-traces/tt-1
 ```
 
+## Fault Injection
+
+### Get Fault Injection
+Request: `GET /meshes/{mesh}/fault-injections/{name}`
+
+Response: `200 OK` with Fault Injection entity
+
+Example:
+```bash
+curl http://localhost:5681/meshes/default/fault-injections/fi1
+```
+```json
+{
+ "type": "FaultInjection",
+ "mesh": "default",
+ "name": "fi1",
+ "sources": [
+  {
+   "match": {
+    "protocol": "http",
+    "service": "frontend",
+    "version": "0.1"
+   }
+  }
+ ],
+ "destinations": [
+  {
+   "match": {
+    "protocol": "http",
+    "service": "backend"
+   }
+  }
+ ],
+ "conf": {
+  "delay": {
+   "percentage": 50,
```

Comment:
Can we pick different value like 50.5 so we show that this is double and not int?